### DISCUSSION TITLE
License not working on macbook pro M2 pro

### DISCUSSION THREAD

**Discussion by marcodiversi** [iterate-ch/cyberduck#14778] [Q&A] [answered]
hello,

I bought a license and on my older Mac it works meanwhile on my new MacBook pro m2 pro I always get the pop-up, it says is registered under my name but the pop-up is showing up, I also sent a support email and it's annoying.
![1](https://github.com/iterate-ch/cyberduck/assets/9339004/488a8f20-8dce-4da6-a9d1-ceaa0dca22ee)
![1](https://github.com/iterate-ch/cyberduck/assets/9339004/21f297df-a51f-4222-b680-bc089982f34c)

I have tried to completely remove it and re-install it again and stuff like that but it's not working! 

thanks

**Comment by Cassie205** [comment, marked as answer]
After activating the license by double-clicking on it the license should be moved to the [application support directory](https://docs.cyberduck.io/cyberduck/support/#application-support-folder). If you still receive the donation prompt, please check if there is a _.cyberducklicense_ file in the application support folder.

**Comment by tatze96** [comment]
Cyberduck can't delete the license file by itself. It can only be deleted by the user or by a third-party application like a security application on your system. Ensure to whitelist the [application support folder](https://docs.cyberduck.io/cyberduck/support/#application-support-folder) if such applications are in use so that Cyberduck works correctly.
